### DISCUSSION TITLE
disqualify cards that exceed price

### DISCUSSION THREAD

**Discussion by thefl0yd** [jef/streetmerchant#1331] [Ideas and feature requests]
While watching output it seems we keep polling cards that exceed our set price:

[3:42:51 PM] info :: ✖ [microcenter] [evga (3080)] ftw3 ultra :: PRICE 809.99 EXCEEDS LIMIT 801

For some retailers, like amazon where various 3rd party sellers can come set prices we probably don't want to do anything - but for standard retailers like Microcenter, Best Buy, etc - we don't expect the price of a given card to fluctuate above and below our set price.

Why not stop repeatedly polling items that are out of our price range? This means less load on both client as well as server and less likelihood of being banned, etc.

**Comment by jef** [comment]
Great question!

> Why not stop repeatedly polling items that are out of our price range? This means less load on both client as well as server and less likelihood of being banned, etc.

Unfortunately, we cannot know the price of the items unless we check the page.
